src/utils/utils.ts

```diff
@@ -0,0 +1,372 @@
+import BN from "bn.js";
+import BigNumber from "bignumber.js";
+import { U64_MAX } from "@solendprotocol/solend-sdk/dist/classes/constants";
+
+export type ReserveSummary = {
+  lendingMarketAddress: string;
+  address: string;
+  symbol: string;
+  mintAddress: string;
+  totalBorrow: BNumber;
+  totalSupply: BNumber;
+  availableAmount: BNumber;
+  decimals: number;
+  cTokenExchangeRate: BNumber;
+  cumulativeBorrowRateWads: BNumber;
+  loanToValueRatio: number;
+  liquidationThreshold: number;
+  price: number;
+};
+
+export class BNumber {
```

Comment:
Oh hmm tbh I really didn't like this BNumber thing we used in the main web repo. I'm guess it was easier to copy some things from the main web repo if we used this? I would prefer switching to BigNumber.js